java/libphonenumber/test/com/google/i18n/phonenumbers/MultiFileMetadataSourceImplTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 The Libphonenumber Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.i18n.phonenumbers;
+
+/**
+ * Unit tests for MultiFileMetadataSourceImpl.java.
+ */
+public class MultiFileMetadataSourceImplTest extends TestMetadataTestCase  {
+
+  protected final MultiFileMetadataSourceImpl multipleMetadataSource;
```

Comment:
Also, looks like the name if left over from a previous class definition? Did you mean multiFileMetadataSource?